ProfileService.js

```diff
@@ -60,7 +59,7 @@ export class ProfileService {
    * @param {object} options - The options to use.
    * @param {string} [options.url = '/profiles-agents'] - The service url to
    *   use.
-   * @param {string} [options.account] - An account ID.
+   * @param {string} [options.account] - An account id.
```

Comment:
I think in documentation `ID` is preferred.

Reply:
I checked `bedrock-account ` and saw ID over id so I made the change: https://github.com/digitalbazaar/bedrock-web-profile/pull/10/commits/45c69e5afb51e161334b90ce60249e23d2bd7db4